File: python/animal.py
class Animal:
    def __init__(self, name, specie):
        self.name = name
        self.specie = specie

    def speak(self):
        print(f"{self.name} says hello to you")

dog = Animal("Yung money", "dog")
cat = Animal("blonedelle", "mehhh")
dog.speak()
cat.speak()

File: python/animal.js
class Animal {
  constructor(name, specie) {
    this.name = name;
    this.specie = specie;
  }
  speak() {
    console.log(`${this.name} says hello to you`);

  }
}

class Dog extends Animal {
  bark() {
    console.log(`${this.name} barks loudly`);
  }
}
const dog = new Dog('Buddy', 'Canis familiaris');
 dog.speak();
dog.bark();
